src/plugins/util/web.js

```diff
@@ -69,6 +69,14 @@ const web = {
     wrapEnd(req)
     wrapEvents(req)
 
+    const enableServerTiming = process.env.SIGNALFX_SERVER_TIMING_CONTEXT
+    if (enableServerTiming && enableServerTiming.trim().toLowerCase() === 'true') {
+      if (!res._sfx_serverTimingAdded) {
+        res.setHeader('Server-Timing', traceParentHeader(span.context()))
+        res._sfx_serverTimingAdded = true
```

Comment:
I think `Object.defineProperty` is preferred to potentially shield this from app logic: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty.